app/views/contexts/hub_context.py

```diff
@@ -42,47 +42,35 @@ class HubContext(Context):
     }
 
     def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
-        hub_schema = self._schema.get_hub()
+        hub_submission_schema = self._schema.get_hub().get("submission", {})
```

Comment:
We should move this to after `if survey_complete:` given this is only needed when survey is complete.